lib/fluent/plugin/cloudwatch/logs/version.rb

```diff
@@ -2,7 +2,7 @@ module Fluent
   module Plugin
     module Cloudwatch
       module Logs
-        VERSION = "0.13.1"
+        VERSION = "0.13.2"
       end
```

Comment:
Please don't touch version constant.

Reply:
I have rolled back version to 0.13.1. Thanks, please review again.